orangecontrib/text/sentiment/__init__.py

```diff
@@ -0,0 +1,71 @@
+import scipy.sparse as sp
+
+from nltk.corpus import opinion_lexicon
+from nltk.sentiment import SentimentIntensityAnalyzer
+from orangecontrib.text import Corpus
+from orangecontrib.text.preprocess import WordPunctTokenizer
+from orangecontrib.text.vectorization.bagofwords import BoWPreprocessTransform, BoWComputeValue
+from orangecontrib.text.vectorization.base import BaseVectorizer
+
+
+class Liu_Hu_Sentiment(BaseVectorizer):
+    positive = set(opinion_lexicon.positive())
+    negative = set(opinion_lexicon.negative())
+    sentiments = ('sentiment')
+
+    def __init__(self):
+        super().__init__()
+        self.dic = {0: 'sentiment'}
+        self.tokens = None
+
+    def _transform(self, corpus, source_dict=None):
+        scores = []
+        if self.tokens is None:
```

Comment:
How would this work if two different corpuses are passed one after another? If I pass the first corpus we would create tokens and store it. Then when I pass another corpus wouldn't the tokens from the first one be used since `self.tokens is not None`?

I wouldn't make `tokens` as an instance attribute. Isn't it only needed inside the `_transform`?